dll/win32/kernel32/client/virtmem.c

```diff
@@ -37,22 +37,17 @@ VirtualAllocEx(IN HANDLE hProcess,
         return NULL;
     }
 
-    /* Handle any possible exceptions */
-    _SEH2_TRY
-    {
-        /* Allocate the memory */
-        Status = NtAllocateVirtualMemory(hProcess,
-                                         &lpAddress,
-                                         0,
-                                         &dwSize,
-                                         flAllocationType,
-                                         flProtect);
-    }
-    _SEH2_EXCEPT(EXCEPTION_EXECUTE_HANDLER)
-    {
-        Status = _SEH2_GetExceptionCode();
-    }
-    _SEH2_END;
+    /* Allocate the memory */
+
+    /* Win2K3 has a SEH just wrap this syscall only,
```

Comment:
This should maybe just be part of the commit log.

Reply:
Yes, I would also omit the comment entirely, and just mention that in the commit comment.

Reply:
Commit log updated